framework/tests/all-systems/java8/lambda/Lambda.java

```diff
@@ -39,6 +48,8 @@ public static void consumeStr(String str) {}
       if (true) return 12;
       else {
         new Lambda(Lambda::consumeStr);           // Use lambda as a constructor argument
+        varargs(new Thread()::start, new Thread()::start);             // Use lambda in a var arg list of method
```

Comment:
One minor note:
It's probably bad form for us to put test cases as fields/initializers and not in a context method if we aren't trying to test fields/initializers specifically.  But I will not remove them now that they are already there.

For this change, the all-systems tests fail for the following reason:
1) They run the initialization checker.
2) When initialization adds defaults to varargs it will add an `@Initialized Lambda this` and expect that varargs is only called after initialization.
3) When varargs is called, initialization has not yet completed so the InitializationChecker will return an error.

I have instead moved the varargs test example to framework/tests/all-systems/java8/lambda/Issue450.java and placed it in a method rather than in an initializer.

And for reference these issues can be caught if you've built the framework by running: 
ant tests-nojdk.

Again, I will wait for any other feedback and then if no one objects to the changes, I will merge this branch.